packaging/make-git-snapshot.sh

```diff
@@ -63,7 +63,6 @@ open(checksum_file, "w").write(json.dumps(j))' $crate_subdir
 # And finally, vendor generated code.  See installdeps.sh
 # and Makefile-rpm-ostree.am for more.
 (cd ${srcdir}
- cp rpmostree-rust{,-prebuilt}.h
```

Comment:
This needs to be dropped in the `tar` command too, a few lines below. CI complained:
```
[2021-03-03T01:20:51.757Z] + tar --transform 's,^,rpm-ostree-2021.2.57.gce28a0a4/,' -rf /home/jenkins/agent/workspace/hub-ci_coreos_rpm-ostree_PR-2631/packaging/rpm-ostree-2021.2.57.gce28a0a4.tar.tmp rpmostree-rust-prebuilt.h rpmostree-cxxrs-prebuilt.h rpmostree-cxxrs-prebuilt.cxx rust/cxx-prebuilt.h

[2021-03-03T01:20:51.757Z] tar: rpmostree-rust-prebuilt.h: Cannot stat: No such file or directory
```

Reply:
Thanks, should be fixed!